analyses/npl/02_glofas_skill.md

```diff
@@ -1,50 +1,30 @@
+Evaluate the return periods, and forecast skill and bias of the GloFAS stations
+
 ```python
 from pathlib import Path
 import os
+import sys
 from importlib import reload
 from collections import Counter
 
 import matplotlib.pyplot as plt
-import matplotlib as mpl
 from matplotlib.ticker import MaxNLocator, ScalarFormatter
-
 import numpy as np
 
-path_mod = f"{Path(os.path.dirname(os.path.realpath(''))).parents[0]}/"
-# chdir because otherwise can't import get_glofas_data
-# should only run this once
-os.chdir(path_mod)
-
-from src.indicators.flooding.glofas import utils, glofas
+import npl_settings as settings
+from src.indicators.flooding.glofas import utils
 from src.utils_general.statistics import calc_mpe
-import src.nepal.get_glofas_data as ggd
-
-reload(utils)
 ```
 
 ```python
-mpl.rcParams['figure.dpi'] = 200
-
-
-COUNTRY_ISO3 = 'npl'
-STATIONS = {
-    'Koshi': ['Chatara_v3', 'Simle_v3', 'Majhitar_v3', 'Kampughat_v3'],
-    'Karnali': ['Chisapani_v3', 'Asaraghat_v3', 'Dipayal_v3', 'Samajhighat_v3'],
-    'Rapti': ['Kusum_v3'],
-    'Bagmati': ['Rai_goan_v3'],
-    'Babai': ['Chepang_v3']
-}
-STATIONS_BY_MAJOR_BASIN = {
-    'Koshi': ['Chatara_v3', 'Simle_v3', 'Majhitar_v3', 'Kampughat_v3', 'Rai_goan_v3'],
-    'Karnali': ['Chisapani_v3', 'Asaraghat_v3', 'Dipayal_v3', 'Samajhighat_v3', 'Kusum_v3', 'Chepang_v3'],
-}
+MAIN_RP = 1.5 # RP used as a threshold to get extreme values
```

Comment:
Would it make sense to put this in the ```settings.py``` file too, or do you think this would change too much between notebooks?

Reply:
I have `MAIN_RP = 2` in the settings file, but in this analysis I want to use 1.5 because the RP Is just used as a cutoff for "high" values, so that's why it's included explicitly here. Maybe there is a better way to make it clear that this is an override?